docker/dev/docker-compose.yml

```diff
@@ -8,10 +12,12 @@ services:
       POSTGRES_PASSWORD: password
       POSTGRES_DB: testnet
     ports:
-      - '5433:5432'
+      - '5432:5432'
```

Comment:
5432 will conflict with rafiki local setup, dsmr fot 3000

Reply:
So should we let rafiki use the defaults and we change testnet itself to 5433 and idk 3003?